lib/versions_repository.py

```diff
@@ -25,3 +25,18 @@ def setup_versions_repository(config):
         raise
 
     return versions_repo
+
+def read_version_and_milestone(versions_repo):
+    """
+    Read current version and milestone (alpha or beta) from VERSION file
+
+    Args:
+        versions_repo (GitRepository): packages metadata git repository
+    """
+    version_path = os.path.join(versions_repo.name, 'VERSION')
+    version_milestone = ""
+    with open(version_path, 'r') as file_version:
```

Comment:
`file_version` looks like we're getting the version of the file. Since it's a file containing the version of the repo, use `version_file` instead.